### PR TITLE
Added new gallery carousel

### PR DESCRIPTION
**Pull request by MDGcode**

### CODE REVIEW

**Review by lseIT — APPROVED** [review]

File: client/src/components/Album.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Carousel } from "react-responsive-carousel";
+import "react-responsive-carousel/lib/styles/carousel.min.css"; // requires a loader
```
Comment (TeoStorm7) [inline client/src/components/Album.js:3]: useless comment

File: client/src/components/Album.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { slides } from "../config/configAlbum";
+export default function Album() {
+  return (
+    <>
+      <Carousel showArrows={true}>
+        {slides.map((slide, slideIndex) => (
+          <div>
+            <img src={slide.url} alt="poza" className=" rounded-xl" />
```
Comment (TeoStorm7) [inline client/src/components/Album.js]: you can put alt content when you mapping

File: client/src/config/configAlbum.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import Poza1 from "../assets/img/webp/gallery/poze-lan-1.webp";
+import Poza5 from "../assets/img/webp/gallery/poze-lan-5.webp";
+import Poza6 from "../assets/img/webp/gallery/poze-lan-6.webp";
+import Poza7 from "../assets/img/webp/gallery/poze-lan-7.webp";
+import Poza8 from "../assets/img/webp/gallery/poze-lan-8.webp";
+import Poza9 from "../assets/img/webp/gallery/poze-lan-9.webp";
+import Poza13 from "../assets/img/webp/gallery/poze-lan-13.webp";
```
Comment (TeoStorm7) [inline client/src/config/configAlbum.js:7]: please suggestions name for your variables

File: client/src/views/Home.js [hunk 4]
```diff
@@ -30,9 +31,9 @@ export default function Home() {
           {/* Banner */}
 
           <div id="Inscrieri" className=" pt-0">
-          <ButoanePrincipale/>
-          <ActivitatiSecundare/>
-          <Cosplay />
+            <ButoanePrincipale />
+            <ActivitatiSecundare />
+            <Cosplay />
```
Comment (TeoStorm7) [inline client/src/views/Home.js:36]: why did you ident here?

File: client/src/views/Home.js [hunk 5]
```diff
@@ -51,10 +52,12 @@ export default function Home() {
 
           {/* Testimoniale */}
 
-          <div id="Galerie" className=" h-3/5">
+          {/* <div id="Galerie" className=" h-3/5">
             <Galery />
+          </div> */}
```
Comment (TeoStorm7) [inline client/src/views/Home.js]: please delete the comment

File: client/src/components/Album.js (hunk 1 above)
Comment (MDGcode) [reply client/src/components/Album.js:3]: ok

File: client/src/components/Album.js (hunk 2 above)
Comment (MDGcode) [reply client/src/components/Album.js]: yes

File: client/src/views/Home.js (hunk 4 above)
Comment (MDGcode) [reply client/src/views/Home.js:36]: Prettify formated

**Review by TeoStorm7 — APPROVED** [review]

**Comment by MDGcode** [comment]
Ms

**Pull request merged**